examples/Cargo.toml

```diff
@@ -8,7 +8,7 @@ publish = false
 identity = { path = "../identity", features = ["account"] }
 pretty_env_logger = { version = "0.4" }
 rand = { version = "0.8" }
-tokio = { version = "1.10", features = ["full"] }
+tokio = { version = "1.14", features = ["full"] }
```

Comment:
Should be `1.15` now. I think all `tokio` dependencies should be updated to use `1.15` in my opinion.